frontend/src/pages/OutfitDetail.tsx

```
@@ -0,0 +1,57 @@
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { getOutfitById, removeClothingFromOutfit, deleteOutfit } from "../api/outfits";
+import type { OutfitDetail } from "../types"; // ✅ Type-only import
+
+const OutfitDetail = () => {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'OutfitDetail'. Consider to delete it or rename it.

'OutfitDetail' is defined here:


(lint/suspicious/noRedeclare)